recipes/cupy-xarray/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cupy-xarray" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cupy-xarray-{{ version }}.tar.gz
+  sha256: fcaf71201fa75bfbb6008740787ee0c34b1ad83d7ca381e2dd915a5e6af7b3c6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - cupy
+    - xarray
+
+test:
+  imports:
+    - cupy_xarray
```

Comment:
On CI, this `import` fails as it is looking for `libcuda.so.1`, but that isn't available. We could...

1. Skip this `import`
2. Force this to build on one of the CUDA images (maybe easier to do in the feedstock)
3. Move this to a `run_test.py` script where exceptions are handled ([like with CuPy]( https://github.com/conda-forge/cupy-feedstock/blob/main/recipe/run_test.py ))

Reply:
Let's just skip the import.

Reply:
Sounds reasonable to me